.gitignore

```diff
@@ -17,10 +17,6 @@ base.db
 /articles-data-test
 /contents-private-test
 /contents-public-test
-fixtures/tuto/article_v1.zip
-fixtures/tuto/article_v1/manifest2.json
-fixtures/tuto/balise_audio.zip
-fixtures/tuto/big_tuto_v1.zip
```

Comment:
je ne vois pas en quoi git te dirait ça. Ces fichiers sont là pour les
tests unitaires de l'import. Ils sont absolument nécessaire.

2016-03-24 10:51 GMT+01:00 Laville Augustin <notifications@github.com>:

> In .gitignore
> <https://github.com/zestedesavoir/zds-site/pull/3414#discussion_r57294062>
> :
>
> > @@ -17,10 +17,6 @@ base.db
> >  /articles-data-test
> >  /contents-private-test
> >  /contents-public-test
> > -fixtures/tuto/article_v1.zip
> > -fixtures/tuto/article_v1/manifest2.json
> > -fixtures/tuto/balise_audio.zip
> > -fixtures/tuto/big_tuto_v1.zip
>
> git me dit toujours que c'est en trop.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3414/files/3b8c76ef52b66c96b2abc09a0228b9c5f80344c7..a0d1a54242e8c5c84eb240bdfd5f443f4273bffb#r57294062>
>